include/openssl/cmp.h.in

```diff
@@ -140,7 +140,6 @@ extern "C" {
 #  if OSSL_CMP_PKIFAILUREINFO_MAX_BIT_PATTERN > INT_MAX
 #   error CMP_PKIFAILUREINFO_MAX bit pattern does not fit in type int
 #  endif
-
```

Comment:
Why this dropped line?

Reply:
I deleted the empty line to make clear that the following decl
```
typedef ASN1_BIT_STRING OSSL_CMP_PKIFAILUREINFO;
```
is directly related.